var/spack/repos/builtin/packages/py-amrex/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyAmrex(PythonPackage, CudaPackage, ROCmPackage):
+    """AMReX Python Bindings with pybind11"""
+
+    homepage = "https://amrex-codes.github.io/amrex/"
+    git = "https://github.com/AMReX-Codes/pyamrex.git"
+
+    maintainers = ["ax3l", "RTSandberg", "sayerhs", "WeiqunZhang"]
```

Comment:
@RTSandberg @sayerhs @WeiqunZhang please let me know if you are ok to be listed as `py-amrex` Spack package co-maintainers. Background: I need someone to approve my PRs to Spack when I update the package (and like to keep the bus-factor >1) :)

Reply:
@etpalmer63 likewise, please let me know if you like to be co-listed :)

Reply:
@ax3l, I recently transferred out of AMReX :cry: , so its probably better to leave me off. Thanks for asking!

Reply:
Sure. Happy to be a co-maintainer.

Reply:
Ok, I can be a co-maintainer